src/backend/gporca/libgpos/server/include/unittest/gpos/common/CSyncHashtableTest.h

```diff
@@ -115,11 +115,7 @@ class CSyncHashtableTest
 		}
 
 		// copy ctor
-		SElem(const SElem &elem)
-		{
-			m_id = elem.m_id;
-			m_ulKey = elem.m_ulKey;
-		}
+		SElem(const SElem &elem) = default;
```

Comment:
Why does copy ctor have different implementation than before changes? It copies m_link now.

Reply:
This constructor is not used. I’ve provided proof—see the last two commits. If it were used implicitly, the tests wouldn’t pass correctly.
I also removed all default constructors, as they are created automatically.